### ISSUE TITLE
Deploy to the organisers.doesliverpool.com VPS

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/weeknote-generator#34]
At present the weeknotes have been running from one of my servers.  Now we're going to be serving images too it *really should* (rather than just *should*) be on DoES infrastructure.

The server being used by `optimism` might as well do double-duty; particularly once #33 is done.

**Comment by amcewen** [comment]
I've set up the DNS for `media.doesliverpool.com`, which points at the `organisers.doesliverpool.com` VPS and set up NGINX to serve files for it (probably still need to sort out the certificates for it).

**Comment by amcewen** [comment]
Ran `certbot --nginx` to update the certificates and get them working for both sites on that VPS.

**Comment by amcewen** [comment]
The `Dockerfile` needs two volumes to be mounted:
* `/var/local/media_root` to hold the published media files.  This should be mounted to `$site_root/m` for the media web server (because I think it'll be useful to put all the content in a top-level folder and something super-short, like `/m/` for Mastodon, seems sensible.
* `/var/local/workfiles` to hold the consent requests and toots waiting to be processed.  This should be mounted to some local folder on the server

**Comment by amcewen** [comment]
Initially I couldn't clone this repo on the production server because it didn't have any keys set up.  Not quite sure why, as https://github.com/DoESLiverpool/optimism/issues/17#issuecomment-914583192 mentions us setting up a deploy key :shrug:  There wasn't a deploy key in `optimism`, so I added it again there.

I was expecting to have to follow [this guide to set up aliases in `ssh`](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/managing-deploy-keys#using-multiple-repositories-on-one-server) and add a deploy key from this repo too, but I could actually just clone it once I'd added the `optimism` deploy key.

I've added cron jobs to look for new posts (and ask for consent) effectively on the hour; and at quarter-past-the-hour it checks for responses to the consent requests.  They both seem to be running okay so far.  I've also set up cron to generate the weeknotes on Monday morning too.  Will leave this open until I've confirmed that that's working.
